### ISSUE TITLE
Orion relationships doesn't work with settinghead:auto-nprogress

### ISSUE THREAD

**Issue by goooseman** [orionjs/orion#202]
Can't figure why, but orion.relationships package doesn't work, when [settinghead:auto-nprogress](https://atmospherejs.com/settinghead/auto-nprogress) is installed. The nprogress spinner is recursively spinning and the meteor can't subscribe to the relationship's publications.
As soon as I removed settinghead:auto-nprogress, everything starts to work as expected.

**Comment by nicolaslopezj** [comment]
No idea. Did you open a issue there?
Do you have any error log?

**Comment by goooseman** [comment]
There are no errors in browser's console or in terminal. If I can get some more errror logs somehow, tell me and I will do it.

**Comment by nicolaslopezj** [comment]
Where auto-nprogress doesn't work, in all routes? just the admin? Only while editing the document?

**Comment by goooseman** [comment]
No, auto-progress works, but the orion relationship plugin can't load other collection to choose from. 
I've created an example to illustrate the issue: [online](http://nprogress-bug-demo.meteor.com/admin) (test@test.com:12345678), [source](https://github.com/goooseman/nprogress-orion-bug-demo).
Just try to create any category: Items dropdown doesn't load at all, but you can see infinite nprogress indicator on top. As soon, as settinghead:auto-nprogress is removed from the project, everything starts to work as expected.
